### PR TITLE
Add app information blurb to welcome page

### LINKED ISSUE #18: Welcome page message
As a visitor, when I visit the welcome page, I see a blurb summarizing the functionality of the GTFO functionality
- [x] Happy/sad path testing

- [x] Update to README (FE screenshots, BE API endpoint documented)

### PR DESCRIPTION
**Pull request by TeknoServal**
This PR adds a welcome blurb to the home page, giving more information and a catchy tagline.

Closes #18 

- [x] All Tests are passing

- [x] Collabotors:
- Aidan Murray
- Nick King

### CODE REVIEW

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -1,6 +1,16 @@
 <h1>Welcome to Get The F(light) Out!</h1>
 
+
+<h3>Gotta go? GTFO.</h3>
+
+<p>
+  Get The F(light) Out! is a travel application built for people who just want
```
Comment (GeorgieGirl24) [inline app/views/welcome/index.html.erb:7]: Great intro!

**Review by GeorgieGirl24 — APPROVED** [review]

**Review by JesseMellinger — APPROVED** [review]
Welcome message looks great!

File: app/views/welcome/index.html.erb (hunk 1 above)
Comment (cpfergus1) [reply app/views/welcome/index.html.erb:7]: I'm stocked by that intro, well done

Comment (stillsheryl) [reply app/views/welcome/index.html.erb:7]: Nice work!

**Review by stillsheryl — APPROVED** [review]

**Review by cpfergus1 — APPROVED** [review]

**Pull request merged**